Library/Formula/corectl.rb

```diff
@@ -28,6 +27,11 @@ def install
     args = []
     args << "VERSION=#{version}" if build.stable?
 
+    if build.head?
+      touch "unreleased"
+      system "git", "add", "unreleased"
+    end
```

Comment:
as i said _cosmetics_. I just wanted to achieve consistency independently the way things are built. just droped that snippet (what it was doing was to enforce the --dirty behaviour from the `git describe` that's in the Makefile).